src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -630,11 +641,12 @@ Parameter Engine::GetMetricLegacy(const std::string& name, const std::map<std::s
     }
 
     IE_CPU_PLUGIN_THROW() << "Unsupported metric key: " << name;
+    OPENVINO_SUPPRESS_DEPRECATED_END
 }
 
-Parameter Engine::GetMetric(const std::string& name, const std::map<std::string, Parameter>& options) const {
-    if (isLegacyAPI())
-        return GetMetricLegacy(name, options);
+ov::Any Engine::get_metric(const std::string& name, const ov::AnyMap& options) const {
```

Comment:
maybe in next pr can consider remove all the metric/config naming function to be more clear for 2.0

Reply:
That is it! This PR only focus on interface changes to minimize changes, of course there will be several following PRs to refactor internal implement.